Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/DeploymentSDK/InvokeRemoteDeployment.ps1

```diff
@@ -0,0 +1,332 @@
+﻿$InitializationScript = {
+    function Invoke-PsOnRemote
+    {
+        param(
+            [string]$machineDnsName,

```

Comment:
addressed in commit [bae6112](https://github.com/mmikkelson/vsts-rm-extensions/commit/bae6112afe5e49f473b944a6fe99992ccbf0df97)